client/my-sites/stats/summary/style.scss

```
@@ -1,8 +1,15 @@
+@import '~@automattic/color-studio/dist/color-variables';
+
 .stats-summary-view {
 	.highlight-cards-heading {
 		@media ( min-width: 661px ) and ( max-width: 1380px ) {
 			margin-left: 32px;
 			margin-right: 32px;
 		}
 	}
+
+	tbody tr:nth-child( odd ),
+	.module-content-table-scroll tbody > tr:nth-child( odd ) > th {
+		background: $studio-gray-0;
```

```suggestion
		background-color: var(--studio-gray-0);
```

We could use existing color variables for the `color-studio` colors.